saphir/src/redirect.rs

```diff
@@ -261,6 +280,10 @@ impl Responder for Redirect {
     fn respond_with_builder(self, mut builder: ResponseBuilder, _ctx: &HttpContext) -> ResponseBuilder {
         builder = builder.status(self.status);
 
+        for (key, value) in self.extra_headers {
```

Comment:
since extra_headers will be of type `HeaderMap<HeaderValue>`, this should be changed for : 
```Rust
builder.headers_mut().map(|hdrs| hdrs.extend(self. extra_headers));
```